public/lib/downloadProbeTest.js

```diff
@@ -58,13 +60,18 @@
        xhr: this._test,
        testRun: this._testIndex
      });
+     var self = this;
+     this.interval = setInterval(function () {
```

Comment:
self is not needed.
this.interval = setInterval(function () {
 this._monitor();
 }.bind(this), 100);